src/compiler/emitter.ts

```diff
@@ -6107,7 +6116,9 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
                         if (namespaceDeclaration && !isDefaultImport(node)) {
                             // import x = require("foo")
                             // import * as x from "foo"
-                            if (!isExportedImport) write("var ");
+                            if (!isExportedImport) {

```

Comment:
The function name (`emitVar`) is misleading and we probably don't need the function at all.

I'd prefer to just have a local in this function `const varOrConst = (languageVersion <= ScriptTarget.ES5) ? 'var ' : 'const ';`) and `write(varOrConst)` at these call sites.